docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -409,14 +394,21 @@ def nabisco_cereals():
     return [1, 2, 3]
 ```
 
-#### To multiple groups
+#### From assets in a sub-module
 
-**Assets can only be assigned to one group at a time**. Attempting to place a grouped asset in a second group will result in an error:
+**This recommended approach** constructs a group of assets from a specified module in your project. Using the `load_assets_from_package_module` function, you can import all assets in a module and apply a grouping:
 
-```bash
-Group name already exists on assets [list_of_asset_keys]
+```python file=/concepts/assets/asset_group_module.py startafter=start_example endbefore=end_example
+import my_package.cereal as cereal
+
+cereal_assets = load_assets_from_package_module(
+    cereal,
+    group_name="cereal_assets",
+)
 ```
 
+If any of the assets in the module already has a `group_name`s explicitly set on it, you'll encounter a `Group name already exists on assets` error.
```

Comment:
Thanks for working this back in!

This looks good, just one thing - I think the `s` after `group_name` is a typo. It looks that way, anyhow:

> already has a `group_name`s